src/Filter/DataFlowFilterType.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+
+namespace App\Filter;
+
+
+use App\Entity\User;
+use Lexik\Bundle\FormFilterBundle\Filter\Query\QueryInterface;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Lexik\Bundle\FormFilterBundle\Filter\Form\Type as Filters;
+use Doctrine\ORM\QueryBuilder;
+
+class DataFlowFilterType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        /** @var User $loggedInUser */
+        $loggedInUser = $options['logged_in_user'];
```

Comment:
Do we really need to be able to pass a user in options? For all practical purposes it will always be the currently logged in user, and then it might be just a easy to get the user in the form type (by injecting `Symfony\Component\Security\Core\Security`).

If we keep the option, I suggest we rename it to `current_user` (or something similar) to make it clear that it may be something else than the currently logged in user.